linker/Mono.Linker/Driver.cs

```diff
@@ -294,6 +296,7 @@ static void Usage (string msg)
 			Console.WriteLine ("                 default is all");
 			Console.WriteLine ("   -x          Link from an XML descriptor");
 			Console.WriteLine ("   -a          Link from a list of assemblies");
+			Console.WriteLine ("   -r          Link from a list of assemblies using public roots");
```

Comment:
I am not sure how useful this really is. We are not doing it on global scale because it can break a lot of code. Perhaps it'd be more useful to do it for all types by default when linker detects it won't break user code